packages/shared/src/components/modals/CommentBox.tsx

```diff
@@ -140,12 +140,20 @@ function CommentBox({
     <>
       <article className="flex flex-col items-stretch rounded-8 break-words-overflow typo-callout">
         <header className="flex items-center mb-2">
-          <ProfilePicture
-            size="large"
-            rounded="full"
-            user={{ image: authorImage, username: authorName }}
-            nativeLazyLoading
-          />
+          {isComment ? (
+            <ProfilePicture
+              size="large"
+              rounded="full"
+              user={{ image: authorImage, username: authorName }}
+              nativeLazyLoading
+            />
+          ) : (
+            <ProfilePicture
+              size="large"
+              user={{ image: authorImage, username: authorName }}
+              nativeLazyLoading
+            />
+          )}
```

Comment:
I think their only difference is the property. I think we can just apply the condition from the property itself. WDYT?
```suggestion
          <ProfilePicture
            size="large"
            rounded={isComment && 'full'}
            user={{ image: authorImage, username: authorName }}
            nativeLazyLoading
          />
```